config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-presubmits.yaml

```diff
@@ -52,7 +47,9 @@ presubmits:
           set -o pipefail;
           set -o xtrace;
           REPO_ROOT=$(git rev-parse --show-toplevel);
-          cd "${REPO_ROOT}/../kubetest2" || exit 1;
-          make install;
-          make install-deployer-gce;
+          cd;
```

Comment:
This feels like it really should be part of `extra_refs`. Is this now the recommended way of handling this case?

Reply:
As I understand it, `extra_refs` clones a git repo, which is what we have been doing. However, this doesn't give us built binaries meaning every test runner using kubetest2 also depends on kubetest2's build dependencies (in this case, the issue arose because the test runner does not have Docker). Using `go get` we get built binaries, avoiding this issue.

Reply:
working directory is your first extra ref, i'm not sure what this CD is for.

Reply:
`cd` in case of modules path weirdness